src/main/java/core/basesyntax/servise/impl/FruitServiceImpl.java

```diff
@@ -0,0 +1,44 @@
+package core.basesyntax.servise.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.servise.FruitService;
+import core.basesyntax.servise.FruitTransaction;
+import core.basesyntax.servise.strategy.OperationStrategies;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class FruitServiceImpl implements FruitService {
```

Comment:
I'm not sure this class adheres to SRP, it does too rather different things, input processing and output generation, consider breaking it up into two services